### PR TITLE
Add canteen recipes.

### PR DESCRIPTION
**Pull request by Rail-Runner**
Add recipes for plastic canteen and 2.5L canteen.

Also remove all the tabs in modified file.

### CODE REVIEW

File: data/json/recipes/recipe_others.json [hunk 1]
```diff
@@ -6624,6 +6624,78 @@
...
+    ]
+  ]
+},{
+  "type" : "recipe",
+  "result": "2lcanteen",
+  "category": "CC_OTHER",
+  "subcategory": "CSC_OTHER_CONTAINERS",
+  "skill_used": "fabrication",
+  "difficulty": 4,
+  "time": 35000,
```
Comment (KA101) [inline data/json/recipes/recipe_others.json]: OK, it takes the same amount of time to make a canteen that's not quite twice as large?

**Comment by KA101** [comment]
If you're gonna astyle, make it a separate commit **at least**.  Huge diff is not helpful.

**Comment by Darkyhard** [comment]
Okay, got it. Should I revert tab removal here?

**Comment by KA101** [comment]
No, you've not got a separate commit so "revert" would be the opposite of useful.

**Comment by KA101** [comment]
Gonna be a bit busy tonight, so no time to check this one.  Sorry.

**Pull request merged**